src/App.jsx

```diff
@@ -1,104 +1,23 @@
 import './App.scss';
+import { PostList } from './components/PostList';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+const postsWithUsersAndComments = postsFromServer.map(post => {
+  const userData = usersFromServer.find(user => user.id === post.userId);
+  const commentsData = commentsFromServer.filter(
+    comment => comment.postId === post.id,
```

Comment:
The `filter` method should not have a comma at the end of the callback function. Remove the comma after `comment => comment.postId === post.id`.